Реализовал 2 л.р.

Namxobick opened the pull request:
Печать всех простых чисел из заданного числового промежутка

KorobejnikovAP commented:
@DydykinPS @klakhov

KorobejnikovAP commented:
@SHVlader

Namxobick committed:
Реализовал 2 л.р.

Namxobick committed:
Закомментировал тест "UpperEqMaxUINT32"

Namxobick committed:
Изменил тип переменной в тестах

Namxobick committed:
рефакторинг

Namxobick committed:
рефакторинг тестов

Namxobick committed:
рефакторинг (cpplint)

Namxobick committed:
Исправил ошибку: out of range

Namxobick committed:
Надеюсь, что пофиксил

Kuznetsov-Artyom commented:
Добавьте пожалуйста описание к вашему пр (чтоб понимать в чем заключалась ваша задача)

Namxobick committed:
оптимизировал + раскомментировал тест

Теперь, если уже считался диапазон с меньшим или равным upperBound, то метод сразу перейдёт в формирование списка простых чисел.

Namxobick committed:
рефакторинг (cpplint)

Namxobick committed:
Удалил тест + добвил reserve

С https://rextester.com/KHJ9839 не поспоришь. Странно, что у меня локально нет такой разницы во времени исполнения (попробовал и с std::chrono::high_resolution_clock).

Kuznetsov-Artyom reviewed (approved):
Спасибо за внесенные изменения

dkurt commented:
@Kuznetsov-Artyom, @FirewallGR, пулл реквест не имеет достаточного числа аппрувов от студентов (практика подразумевает что студенты оставляют комментарии друг другу тоже). Пожалуйста дожидайтесь когда бот выставляет метку "Ready for merge"
